draft-ietf-quic-http.md

```diff
@@ -767,14 +767,20 @@ indicated in the GOAWAY frame, those requests were not and will not be
 processed.  Endpoints SHOULD reset any streams above this ID with the error code
 HTTP_REQUEST_CANCELLED.  Servers MAY also reset streams below the indicated ID
 with HTTP_REQUEST_CANCELLED if the associated requests were not processed.
+Servers MUST NOT use the HTTP_REQUEST_CANCELLED status for requests which were
+partially or fully processed.
```

Comment:
Clients can detect this by observing that it received part of a response. Do you want to recommend a reaction for clients?

Reply:
Following paragraph discusses what clients do with cancelled requests and this PR adds, "However, if a stream is cancelled after receiving a partial response, the response SHOULD NOT be used and the request cannot be safely retried."

Though rereading it, it might be clearer to swap the order of the last two sentences.